websites/Q/Q-dance/presence.ts

```diff
@@ -1,27 +1,24 @@
-var presence = new Presence({
-    clientId: "619768959717343242"
+const presence = new Presence({
+    clientId: "844107169205190686"
   }),
   strings = presence.getStrings({
     live: "presence.activity.live"
   });
-
-var elapsed = Math.floor(Date.now() / 1000);
+   let elapsed = Math.floor(Date.now() / 1000);
```

Comment:
let -> const and merge

Reply:
still not resolved

Reply:
![image](https://user-images.githubusercontent.com/42322979/119102534-6c128080-ba1a-11eb-9a00-801d7fa2857d.png)

Reply:
i don't see any sense in this code since elapsed can't be null tho 🤔

Reply:
.

Reply:
still not fixed.